### ISSUE TITLE
Using std::chrono::duration_cast results in an error

### ISSUE THREAD

**Issue by tenk-a** [LADSoft/OrangeC#49]
Using std::chrono::duration_cast results in an error

smp.cpp:
```
#include <chrono>
using namespace std::chrono;
int main() {
    system_clock::time_point b;
    system_clock::time_point e = system_clock::now();
    microseconds             d = duration_cast<microseconds>(e-b);
}
```
RESULT:
Error(113)    C:\Program Files (x86)\Orange C 386\include\chrono(389):  Call of nonfunction in function std::__1::chrono::duration_cast<duration<long long, ratio<long long=1, long long=1000000>>, long long, ratio<long long=1, long long=1000000>>( const duration<long long, ratio<long long=1, long long=1000000>>&)
Warning(436)  smp.cpp(6):  In template instantiation started here

**Comment by LADSoft** [comment]
apparently the fix for the bind issue in #46 contributed to breaking this.   Fixed this issue and reworked that one.

**Comment by tenk-a** [comment]
I compiled smp.cpp with 6.0.31.1
Thank you.
